Catalog.html

```diff
@@ -0,0 +1,362 @@
+            <a class="navigation-link" href="gostinica.html">Гостиницы</a>
+          </li>
+        </ul>
+        <img class="logo" src="#" alt="логотип" />
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden"> Поиск </span>>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden"> Избранное </span>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden"> Заказать </span>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-inner">
+      <header class="inner-header">
+        <h1 class="inner-header-title">Гостинницы Седоны</h1>
+        <ul class="breadcrumbs">
+          <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link" href="index.html">Главная</a>
+          </li>
+          <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link">Каталог</a>
+          </li>
+        </ul>
+      </header>
+      <section class="catalog-products">
+        <h2 class="visually-hidden">Список гостиниц Седоны</h2>
+        <form
+          class="catalog-filter"
+          action="https://echo.htmlacademy.ru/"
+          method="get"
+        >
+          <fieldset class="infrastructure-filter-group">
+            <legend class="infrastructure-filter-title">Инфраструктура</legend>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="swimming pool"
+              />
+              <span class="control-mark"></span>
+              <span class="control-label">Бассейн</span>
+            </label>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="parking space"
+              />
+              <span class="control-mark"></span>
+              <span class="control-label">Парковка</span>
+            </label>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="Wi-fi"
+              />
+              <span class="control-mark"></span>
+              <span class="control-label">Wi-fi</span>
+            </label>
+          </fieldset>
+          <fieldset class="housing-filter-group">
+            <legend class="housing-filter-title">Тип жилья</legend>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="hotel"
+              />
```

Comment:
В этом филдсете радиокнопки. Чтобы они работали корректно, нужен один и тот же `name`, но разные `value`. Первый элемент по умолчанию `checked`
```suggestion
              <input
                class="visually-hidden control-input"
                type="radio"
                name="category"
                value="Гостиница"
                checked
              />
```